### ISSUE TITLE
Add 'The Art of Teaching' blog post

### ISSUE THREAD

**Issue by carpodaster** [rails-girls-summer-of-code/summer-of-code-2014#169]
Will expand on the coaches guide as an additional resource and to keep the original coaches' guide short and concise.

**Comment by anikalindtner** [comment]
@carpodaster you are taking care of this, right? when did you plan on publishing it?

**Comment by carpodaster** [comment]
Yeah, I'd like to scribble something for this. Will take a while, though - I'm thinking next weekend or something. If anyone wants to have it faster than that: please go ahead.

**Comment by sareg0** [comment]
@carpodaster if you want anyone to take a look at it just let me know.

**Comment by anikalindtner** [comment]
hej @carpodaster: i think in the next weeks would be a good time to publish something like this. can you work on that? do you need help?

**Comment by carpodaster** [comment]
Oh dear, this has been idling for some time now. Thanks @sareg0 and @anikalindtner for offering support.

I have several hours on a train this weekend so let's see how far that gets me :smile:

**Comment by sareg0** [comment]
@carpodaster did this one get posted or written up in the end? We can still put it on the editorial calendar, and have it published after the summer of code, so we have regular updates coming out after the program.

**Comment by carpodaster** [comment]
Nah, I slacked on this one. Hard. Although I indeed did some investigation and interviewed a teacher-friend :smile:  I would still love to do it and posting somewhen after the October sounds great.

**Comment by alicetragedy** [comment]
+1 for publishing after the Summer of Code so the blog isn't too boring after the summer is over :sunglasses:
